cardano-db-sync/src/Cardano/DbSync.hs

```diff
@@ -141,7 +141,7 @@ runDbSync metricsSetters knownMigrations iomgr trce params aop = do
 extractSyncOptions :: SyncNodeParams -> Bool -> SyncOptions
 extractSyncOptions snp aop =
   SyncOptions
-    { soptExtended = enpExtended snp
+    { soptExtended = enpExtended snp && enpHasCache snp
```

Comment:
does this mean that we don't do epoch calculations when the no cache flag is on?
If we do still do calculations then could this be it's own value then I can use it directly in the epoch PR